Makefile

```diff
@@ -38,6 +38,7 @@ run-local:
 	SAAS_BASE_URL="https://meshery.layer5.io" \
 	PORT=9081 \
 	DEBUG=true \
+	ADAPTER_URLS="mesherylocal.layer5.io:10000 mesherylocal.layer5.io:10001" \
```

Comment:
Are we missing 10002? What is ADAPTER_URLS used for?

Reply:
Purely for local dev. This is different from the one used in docker-compose.